src/backend/model/personalListModel.tsx

```diff
@@ -1,6 +1,49 @@
-import { atom, useRecoilState } from "recoil";
-import { fetchSearchResults } from "../api/imdb/IMDB";
-const myList = atom({
+import { atom, useRecoilState, useRecoilValue, useSetRecoilState } from "recoil";
+import { fetchTitle } from "../api/imdb/IMDB";
+import {fetchAvailability } from "../api/availability/streamingAvailability"
+import { Stargate } from "../../backend/model/dummyStargate";
+import availability from "../../backend/model/streamingAvailabilityDummyStargate";
+import react from "react"
+
+const dummyFirebaseMyList = ["tt0118480", "tt0118480"]
+ 
+const myListState = atom({
   key: "myList", // unique ID (with respect to other atoms/selectors)
   default: [], // default value (aka initial value)
 });
+export const selectedSeasonState = atom({
```

Comment:
I'm going to need this in the search results view too. I already added an atom to src/atoms.tsx but it's better if we keep it in the model. Should we put this in a shared file instead?

Reply:
Hm I guess we could put all the atoms in their own file and just move it to the model. Doesn't matter much to me if it is in this file or the atom file